src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -229,7 +229,11 @@ async function minikubeStatus(context: Context): Promise<MinikubeInfo> {
     const result = await context.shell.exec(`"${context.binPath}" status`);
 
     if (result && result.stderr.length === 0) {
-        const hostStatus = result.stdout.split('\n')[0].split(': ')[1].toLowerCase();
+        const hostItem = result.stdout.split('\n').find((status) => status.includes('host'));
```

Comment:
When I run `minikube status` on my home machine, I get:

```
C:\Users\ivan> minikube.exe status
�🤷  There is no local cluster named "minikube"
�👉  To fix this, run: minikube start
```

This doesn't contain a `host` line, but it feels like it should return `running: false` rather than failing?